Jenkinsfile

```diff
@@ -1361,4 +1250,9 @@ pipeline {
             }
         }
     }
+    post {
+        unsuccessful {
+            notifyBrokenBranch branches: "master PR-1068"
```

Comment:
Not sure I understand the PR-1068 part

Reply:
Testing code.  Removed in the [latest commit](https://github.com/daos-stack/daos/pull/1068/commits/ccaa657a115a337d9e318248be902bb33b16bfa0).